aiida_testing/mock_code/_fixtures.py

```diff
@@ -48,26 +106,58 @@ def _get_mock_code(
             stored.
         ignore_files :
             A list of files which are not copied to the results directory
-            when the code is executed.
+            after the code has been executed.
+        executable_name :
+            Name of code executable to search for in PATH, if configuration file does not specify location already.
+        config :
+            Dict with contents of configuration file
+        config_action :
+            If 'require', raise ValueError if config dictionary does not specify path of executable.
+            If 'generate', add new key (label) to config dictionary.
+        generate_config :
```

Comment:
The `generate_config` parameter doesn't exist, and `regenerate_test_data` is missing in the docstring.

Reply:
thanks, fixed!